src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -87,7 +87,10 @@ public MetadataProvider(Docset docset, Cache cache, MicrosoftGraphCache microsof
                 }
             }
 
-            errors.AddRange(_schemaValidator.Validate(result));
+            if (file.ContentType == ContentType.Page && string.IsNullOrEmpty(file.Mime))
```

Comment:
Is it correct we hardcode the ContentType here?
I see validation rule includes the type such as conceptual, include, is it the same thing?
https://docsmetadatatool.azurewebsites.net/rules

Reply:
this makes it validating against ONLY 'Conceptual'(not other yaml schemas), is this what we want?

Reply:
@kimizhu , I think we will support different rules per types eventually, but we can start with just conceptual content. docfx today have some small problems identifying if a markdown is a include file (token)

Reply:
In order to be consistent with v2, currently we only verify markdown file

Reply:
@yufeih what is the dev effort to implement the ability to identify the include files?
The problem here is there are some rules only applies to the conceptual content - for example, title is a required metadata for conceptual file but not required for include file. If docfx v3 cannot tell if the file is include file or not, we cannot apply this rule only on conceptual files.